packages/synapse-react-client/src/components/CreateTableViewWizard/CreateTableViewWizard.tsx

```diff
@@ -124,6 +126,28 @@ export default function CreateTableViewWizard(
       },
     })
 
+  const {
+    mutateAsync: validateSql,
+    data: isSqlValid,
+    error: sqlValidationError,
+  } = useValidateDefiningSql(sql, entityType!, {
+    onSuccess: (isSqlValid: ValidateDefiningSqlResponse) => {
+      if (isSqlValid && isSqlValid.isValid) {
+        setStep('TABLE_NAME')
+      }
+    },
+  })
+
+  const showSqlValidationError: string | null = useMemo(() => {
```

Comment:
Since it is a string, suggest renaming
```suggestion
  const sqlValidationError: string | null = useMemo(() => {
```